components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -25,94 +25,229 @@ interface Props {
-  grid-template-columns: 16px 1fr;
-  padding: 0 8px;
-  gap: 8px;
-  color: #339AF0;
+const ControlsContainer = styled.span`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const InfoIcon = styled.span`
+  position: relative;
+  margin: 0px 18px;
+
+  > .icon {
+    height: 17px;
+    width: auto;
+    vertical-align: middle;
+  }
+
+  .tooltip {
+    position: absolute;
+    bottom: 100%;
+    left: -200px;
+    width: 264px;
+    padding-bottom: 12px;
+    display: none;
+  }
+
+  &:hover .tooltip {
+    display: initial;
+  }
+`
+
+const InfoIconTooltip = styled.div`
+  position: relative;
+  background: var(--background1);
+  box-shadow: 0px 0px 24px rgba(99, 105, 110, 0.36);
+  border-radius: 6px;
   font-family: var(--brave-font-family-non-serif);
   letter-spacing: 0.01em;
-  margin-bottom: 7px;
-`
+  padding: 24px;
 
-const DescriptionIcon = styled.div`
-  background: url(${InfoIcon}) no-repeat;
-  background-size: 16px 16px;
-  background-position: center;
+  &:before {
+    content: '';
+    position: absolute;
+    bottom: -7px;
+    left: 201px;
+    background: inherit;
+    height: 15px;
+    width: 15px;
+    transform: rotate(45deg);
+  }
 `
 
-const DescriptionTitle = styled.div`
+const InfoIconTooltipTitle = styled.div`
   font-weight: 600;
+  font-size: 14px;
+  line-height: 24px;
+  margin-bottom: 8px;
+`
+
+const InfoIconTooltipBody = styled.div`
+  font-weight: 400;
   font-size: 12px;
   line-height: 18px;
 `
 
+const Container = styled.div`
+  background-color: var(--info-background);
+  border-radius: 8px;
+  padding: 24px 24px;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin-top: 8px;
+`
+
+const DescriptionRow = styled.div`
+  width: 100%;
+  color: var(--text1);
+  font-family: var(--brave-font-family-non-serif);
+  letter-spacing: 0.01em;
+  margin-bottom: 16px;
+`
+
+const DescriptionTitle = styled.div`
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 24px;
+  margin-bottom: 4px;
+`
+
 const DescriptionBody = styled.div`
-  grid-column: 2;
-  font-weight: 500;
-  font-size: 11px;
-  line-height: 17px;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
 `
 
 const EnableRewardsButton = styled.button`
-  width: 236px;
-  height: 40px;
+  height: 36px;
   background: var(--interactive5);
-  border-radius: 48px;
+  border-radius: 1000px;
   border-width: 0;
   color: white;
   cursor: pointer;
-  margin-bottom: 8px;
+  padding: 10px 20px;
   font-family: var(--brave-font-family-non-serif);
   font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  align-self: start;
 `
 
-export default function SponsoredImageToggle ({ onChange, onEnableRewards, checked, disabled, rewardsEnabled: rewardEnabled, adsEnabled, canSupportAds }: Props) {
-  const showRewardButton = checked && !disabled && (!rewardEnabled || (!adsEnabled && canSupportAds))
+function GetInfoTooltipText (checked: boolean, rewardsEnabled: boolean) {
```

Comment:
Nit: I think functions should be lower case (except for components) to fit in with most other stuff. This is pretty inconsistently enforced though (sorry)

```suggestion
function getInfoTooltipText (checked: boolean, rewardsEnabled: boolean) {
```